ipapython/ipautil.py

```diff
@@ -1294,15 +1294,17 @@ def unescape_seq(seq, *args):
 
 def escape_seq(seq, *args):
     """
-    escape (prepend '\\') all occurences of sequence in input strings
+    escape (prepend '\\') all occurences of sequence in input strings unless it
+    is already escaped.
```

Comment:
I think that the function called `escape_seq` should have no say in what is being escaped. Given that next commit modified `get_directive` to be inverse of `set_directive`, why is this change necessary at all?